Doc/source/coordinates.rst

```diff
@@ -1,32 +1,99 @@
+geodetic (GDZ), which is defined in [altitude, latitude, longitude]
+where altitude is relative to a reference ellipsoid. Similarly, distance
+units are assumed to be Earth radii (Re) in all systems except GDZ, where
+altitude is given in km. Conversions to GDZ will output altitude in km
+regardless of the input distance units and conversions from GDZ will
+output in Re regardless of input units. In all other cases, the distance
+units will be preserved.
 
-.. autosummary::
-   :template: clean_function.rst
-   :toctree: autosummary
+    .. versionchanged:: 0.3.0
 
-    car2sph
-    sph2car
-    quaternionRotateVector
-    quaternionNormalize
-    quaternionMultiply
-    quaternionConjugate
-    quaternionFromMatrix
-    quaternionToMatrix
+    The new CTrans backend was added, which includes support for the names
+    ``ECI2000``, ``ECIMOD``, ``ECITOD``, and ``CDMAG``. With the exception
+    of ``ECIMOD``, these can be used with the existing IRBEM backend, and
+    will be converted to their closest equivalents.
+
+    .. versionchanged:: 0.4.0
+
+    The default backend for coordinate transformations was changed from IRBEM
+    to the CTrans-based SpacePy backend.
+
+differences between representations
+--------------------------------------------
+IRBEM's coordinate transformations are low-accuracy and were written for
+a library with a driving philosophy of speed and robustness as priorities.
+The coordinate transformations are therefore approximate. Further, most of
+the geophysical systems (e.g., GSE, SM) are derived from an inertial
+system. It is standard practice to use ECIMOD as this system. However,
+IRBEM does not currently make ECIMOD available as one of its inertial
+systems. IRBEM's default inertial system (called GEI) is consistent with
+an approximation of ECITOD. Hence there will be small differences between
+IRBEM's transformations and those using SpacePy's CTrans backend.
+Further sources of difference include: IRBEM uses a low-order approximation
+to the sidereal time and other parameters; the calculation of the Earth-Sun
+vector differs between the representations; the definitions of an Earth
+radius differ (SpacePy = 6378.137km; IRBEM = 6371.2 km). SpacePy's in-built
+representation is higher accuracy and is comprehensively tested, including
+tests for consistency with other high accuracy packages such as LANLGeoMag
+and AstroPy. However, for use cases where the required precision is of order
+1 percent the output can be considered equivalent.
+
+Setting options for coordinate transformation
+---------------------------------------------
+The backend for coordinate transformations can be provided at
+instantiation of a :class:`Coords` object using a keyword
+argument. However, for convenience and flexibility the options can be
+set at the module level. Configurable options include the backend used
+(:mod:`~spacepy.irbempy` or SpacePy's :mod:`~spacepy.ctrans`) and the
```

Comment:
A lot of these probably do not need the `:class:` etc.; we have it set to auto-search. Similarly you may be able to get away with just `irbempy`. Do the minimal markup that works, here and elsewhere, so it's a bit more readable if looking at source.

Reply:
Oddly enough, either way I do it does not result in a hyperlink being generated.

Reply:
This is just a reminder to myself to add nitpicky when this is the last comment left, I will commit the current module stuff now, and amend later with the conf.py edits.